doc/intro.md

```diff
@@ -126,8 +126,8 @@ Curve Cryptography. `r*G + v*H` is called a _Pedersen Commitment_.
 As an example, let's assume we want to build a transaction with two inputs and one
 output. We have (ignoring fees):
 
-* `vi1` and `vi2` as input values.
-* `vo3` as output value.
+* *vi1* and *vi2* as input values.
```

Comment:
For consistency sake we should use either \*x\* or \`x\` for mathematical notation.

Reply:
Same as \_x\_ which is equivalent to \*x\*

Reply:
The convention used in the document seems to be that variable names are in italics while mathematical expressions are code-formatted. I simply followed that convention.

Reply:
Okay 👍